localization/replace/german/dlc/ep1/dlc_ep1_fund_inspiration_events_l_german.yml

```diff
@@ -2,47 +2,47 @@
 
 ## Postcards from your Adventurer by Isabella Welch
 
 #Cyrodiil
 
- fund_inspiration.6001.t:0 "Cyrodiil and saddle gallop through my heart,"
- fund_inspiration.6001.desc:0 "I have been riding for months on end. My rear shall never recover! I've acquired quite the taste for Imperial Stout and nights beneath the open sky.\n\nWith this courier, I send a Varla Stone — a stone created by the Ayleids, supposed to be made from something out of Aetherius. Perhaps you have a use for it."
- fund_inspiration.6001.a:0 "I'll hold onto this!"
- fund_inspiration.6001.b:0 "Send it to the Court Mage."
+ fund_inspiration.6001.t:0 "Reise durch Cyrodiil ,"
+ fund_inspiration.6001.desc:0 "Ich bin monatelang unterwegs gewesen. Mein Hintern wird sich nie wieder erholen! Ich habe eine Vorliebe für Kaiserliches Starkbier und Nächte unter freiem Himmel entwickelt.\n\nMit diesem Kurier sende ich einen Varla-Stein - einen von den Ayleiden geschaffenen Stein, der aus etwas aus Aetherius hergestellt sein soll. Vielleicht habt Ihr eine Verwendung dafür"
+ fund_inspiration.6001.a:0 "Ich werde es aufbewahren!"
+ fund_inspiration.6001.b:0 "Schickt es an den Hofmagier."
 
 #Skyrim
 
- fund_inspiration.6002.t:0 "My jaunt through Skyrim marches on!"
- fund_inspiration.6002.desc:0 "I am near-blinded by all the snow in the wilderness. I find myself yearning for the warmth of a hearth.\n\nI have sent a courier with an urn I foundan urn I found. Have you ever seen something so garish?"
- fund_inspiration.6002.a:0 "I rather like it, it's [inspiration_owner.GetHerHis] loss."
- fund_inspiration.6002.b:0 "Ghastly! Take it apart and sell the gems."
+ fund_inspiration.6002.t:0 "Mein Streifzug durch Himmelsrand geht weiter!"
+ fund_inspiration.6002.desc:0 "Der Schnee in der Wildnis hat mich fast geblendet. Ich sehne mich nach der Wärme eines Herdes.\n\Ich habe einen Kurier mit einer Urne geschickt, die ich gefunden habe. Habt Ihr jemals etwas so Grelles gesehen?"
```

Comment:
Der Schnee in der Wildnis hat mich fast geblendet. Ich sehne mich nach der Wärme eines Herdes.\n\Ich habe einen Kurier mit einer Urne geschickt, die ich gefunden habe. Habt Ihr jemals etwas so schrilles gesehen?